config/schema/format_strawberryfield.schema.yml

```diff
@@ -614,3 +614,14 @@ field.formatter.settings.strawberry_citation_formatter:
         label: 'style'
     localekey:
       type: string
+condition.plugin.ado_type_condition:
+  type: condition.plugin
+  label: 'ADO Type Condition'
+  mapping:
+    ado_types:
+      type: array
+      label: 'ADO Type'
+    recurse_ado_types:
+      type: boolean
```

Comment:
And you are not saving booleans but 0 and 1, so you either make it an integer OR you cast to boolean before saving in your code (which is basically adding a (bool) and also making sure maybe? that the default is a bool?

Reply:
Updated.